scripts/template_docsting_with_example.py

```diff
@@ -0,0 +1,77 @@
+# see more documentation guidelines online here: https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)
+
+
+class SomeClass:
+    """A one line description of the Class.
+
+    A short paragraph providing more details.
+
+    Deprecation warning
+
+    Extended Summary
+
+    Parameters
+    ----------
+    scale_factor :class:`float`
+        The factor used for scaling.
+
+    Returns
+    -------
+    :class:`VMobject`
+        Returns self.
```

Comment:
I know that this is "just" a template, but personally I like to avoid to mention ``self`` in context like this. It is rather technical terminology, and I think that something like

```suggestion
        Returns the modified :class:`~.VMobject`.
```

is easier to understand. (However, just as mentioned, this is a personal preference.)